frontend/src/index/index.scss

```diff
@@ -76,9 +76,38 @@ code {
     stroke-opacity: 0.5;
 }
 
-.map-square-box {
-    stroke: steelblue;
+//.map-square-box {
+//    stroke: steelblue;
+//    stroke-width: 2.8;
+//    fill: dodgerblue;
+//    fill-opacity: 0.3;
+//}
+.map-square {
     stroke-width: 2.8;
-    fill: dodgerblue;
-    fill-opacity: 0.3;
+    fill-opacity: 0.5;
+}
```

Comment:
It looks like this `.map-square` class isn't getting applied anywhere. It turns out that the default styling supplied to the `<Rectangle>` element by our external library almost perfectly matches the styling supplied by this class. You can investigate that through the `Elements` tab of the Chrome Dev Tools. The consequence is that it's not visually apparent that this class isn't actually doing anything! We can fix resolve this by adding it to the string we assign in `mapSquareBucket`.

![Screen Shot 2021-04-06 at 4 01 55 PM](https://user-images.githubusercontent.com/20114078/113771533-7b9b6b80-96f1-11eb-841b-7d2b04657b73.png)